### ISSUE TITLE
[Suggestion]A text segment that differ from the default language is not programmatically identified or described

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1922]
Tools: Web Accessibility Toolbar (WAT2.0) 

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Lists tab.
3. Use the WAT (Doc Info - Show Lang Attributes) to reveal the language attribute.
4. Check default language of the page and any passages that differ from the default language.

Actual Results:
A text segment on the page is not in English, but the change in language is not identified programmatically. Screen reader will read this segment in English.
For more details, please refer to “LanguageIsDifferent.png”.

Expected Results:
There is an accurate language attribute set for each text segment that is different from the page's default language. Then screen reader will read correctly.
Or describe this text segment as an example.

Violation of Section 508 Standards: 
Section 1194.31(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided. 
Section 1194.31(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided. 
User Impact: Blind.

Screen shot:
1.	LanguageIsDifferent.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/0f97a4b8-d67d-4255-babb-e8f726549f91)

**Comment by mathisscott** [comment]
This is not an issue. This is Lorem Ipsum placeholder text.
